src/auth/index.ts

```diff
@@ -48,20 +49,23 @@ router.post(
       .not()
       .isEmpty(),
   ],
-  (req: Request, res: Response, next: NextFunction) => {
+  (req: Request, res: Response, next: NextFunction): void => {
     const errors = validationResult(req);
```

Comment:
We can use your middleware `handleRequestError` function here.

And we could reorganise the `auth` request handlers in the same manner as you did for the `api` endpoints.